### PR TITLE
Made Additional Changes

### PR DESCRIPTION
**Pull request by vpritamani-rubrik**
Changes include:
- Azure Backup Costs
- Anonymization Feature
- Improving consolidation script
- Separate MI SQL and SQL, added LTR/STR data for MI SQL
- Fixing potential region issues giving commas instead of .'s in CSV output

### CODE REVIEW

File: CLOUD/Get-AzureSizingInfo.ps1
Comment (DamaniN) [inline CLOUD/Get-AzureSizingInfo.ps1:1]: In the `azure_mi_db_items` JSON files the `-Anonymize` function must anonymize the following fields:

- Id
  - Leave the path intact only anonymize the subscription ID, resource group ID, managed instance name and database name. 
  - The IDs and names should match the corresponding file. 
- ResourceGroupName
- DatabaseName
- ManagedInstanceName
- Name
- InstanceName

Comment (DamaniN) [inline CLOUD/Get-AzureSizingInfo.ps1:1]: The `azure_vmdisk_info` CSV file is corrupted. I'll DM you an example of what's in it.

**Review by DamaniN — CHANGES_REQUESTED** [review]
There are a few issues to fix. Review the comments on each file.

File: CLOUD/Get-GCPSizingInfo.ps1
[the remark this answers is not in the document]
Comment (vpritamani-rubrik) [reply CLOUD/Get-GCPSizingInfo.ps1:1]: This only happens when the archive file is the same name - as the archive file is named based on the hour and minute it runs, this 'error' only occurs if someone tries to run the script multiple times in the same minute, which shouldn't happen

[the remark this answers is not in the document]
Comment (vpritamani-rubrik) [reply CLOUD/Get-GCPSizingInfo.ps1:1]: I implemented this but with regular numbers and not padded numbers

File: CLOUD/Get-AWSSizingInfo.ps1
[the remark this answers is not in the document]
Comment (vpritamani-rubrik) [reply CLOUD/Get-AWSSizingInfo.ps1:1]: For the ARNs, the problem with AWS is that these are not consistent across different resource types/workloads. One can see these inconsistencies in how I have to process the ARNs when assigning backup plans to resources.

File: CLOUD/Get-GCPSizingInfo.ps1
[the remark this answers is not in the document]
Comment (vpritamani-rubrik) [reply CLOUD/Get-GCPSizingInfo.ps1:1]: Will add seconds

[the remark this answers is not in the document]
Comment (vpritamani-rubrik) [reply CLOUD/Get-GCPSizingInfo.ps1:1]: Will make the anonymize tag exclude the output log from the ZIP file, keep it in the system. Add note for customer to sanitize output log before sending.

Create a output log in ZIP to just say "Anonymized file; customer has original"

**Comment by vpritamani-rubrik** [comment]
@DamaniN Fixed all of the issues incorporating feedback from our talk yesterday

**Review by DamaniN — APPROVED** [review]
Logged a few issues in our internal tracker to push out this PR. All other changes look good.

**Pull request merged**